build.gradle

```diff
@@ -1006,6 +1009,33 @@ project(':iceberg-open-api') {
     recommend.set(true)
   }
   check.dependsOn('validateRESTCatalogSpec')
+
+  // Create a custom configuration that extends testFixturesImplementation and is resolvable
+  configurations {
+    testFixturesShadowImplementation {
+      canBeResolved = true
+      canBeConsumed = false
+      extendsFrom testFixturesImplementation
+    }
+  }
+
+  shadowJar {
+    archiveBaseName.set("iceberg-open-api-test-fixtures-runtime")
+    archiveClassifier.set(null)
+    configurations = [project.configurations.testFixturesShadowImplementation]
```

Comment:
can't directly depend on `testFixturesImplementation` as it is not resolved. Hence using via custom `testFixturesShadowImplementation`